hedera-node/hedera-mono-service/src/main/java/module-info.java

```diff
@@ -94,7 +94,8 @@
     exports com.hedera.node.app.service.mono.context.properties;
     exports com.hedera.node.app.service.mono.state.enums to
             com.hedera.node.app.service.mono.test.fixtures,
-            com.hedera.node.app.service.token.impl.test;
+            com.hedera.node.app.service.token.impl.test,
+            com.hedera.node.services.cli;
```

Comment:
We're exposing the `...state.enums` package to more than just tests here – now it's more 'public' than before, more a part of the API. This may not be an issue, but we may want to ask the powers that be.

Reply:
Sounds good to me 👍